.github/workflows/commit-check.yml

```diff
@@ -20,16 +20,22 @@ jobs:
   check-commit-message:
     name: Check Commit Message
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14]
+        fetch-depth: [2]
```

Comment:
using a matrix means there will be more then one option provided. Here, that is not true. Both are static and we won't be changing that.

Reply:
Reverted, thanks